### PR TITLE
Fix the issue 10281: Readme content of Storage Queue is unclear and inconsistencey

### PR DESCRIPTION
**Pull request by Melanie-Ma**
Fix the issue https://github.com/Azure/azure-sdk-for-java/issues/10281

### CODE REVIEW

File: sdk/storage/azure-storage-queue/README.md [hunk 1]
```diff
@@ -173,7 +173,8 @@ queueClient.createWithResponse(metadata, null, Duration.ofSeconds(30), Context.N
 or
 
 ```Java
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-queue/README.md:175]: @gapra-msft @rickle-msft we should update this to use embedme.

Comment (Melanie-Ma) [reply sdk/storage/azure-storage-queue/README.md:175]: There is no `ReademeSample.java` and no `embedme` comments

Comment (Melanie-Ma) [reply sdk/storage/azure-storage-queue/README.md:180]: @alzimmermsft - I have updated this to use embedme,  please review.

**Comment by Melanie-Ma** [comment]
@alzimmermsft - Hi, can you help me to fix the error of ci build? Thanks.
image

**Comment by alzimmermsft** [comment]
Hi @Melanie-Ma, the error you posted is a bit of a red herring. Looking at the end result of the build shows that the portion of the build that failed was `azure-storage-queue`.

image

It failed during the README codesnippet validation. Unfortunately, it appears the error doesn't get logged in red text making it clear that it was the reason the build failed.

image

Run this command to generate the codesnippets the validation is expecting, you may need to change target line numbers or code to ensure it is what you want.

```
mvn clean package -Dreadme-codesnippet-windows -DskipTests -f .\sdk\storage\azure-storage-queue\pom.xml
```

**Pull request merged**